<?xml version="1.0" ?>
<launch>
    <group ns="cslibs_ndt_2d">
        <param name="path_ndt" 	   value="/data/0/rawseeds/maps/with_stereo/Bicocca_2009-02-25a-dynamic-natural-light-groundtruth-cartographer/map/2d/ndt/map" />
        <param name="path_occ_ndt" value="/data/0/rawseeds/maps/with_stereo/Bicocca_2009-02-25a-dynamic-natural-light-groundtruth-cartographer/map/2d/occ_ndt/map" />

	<param name="ndt_sampling_resolution" 	  value="0.025"/>
	<param name="occ_ndt_sampling_resolution" value="0.025"/>
    </group>

    <node name="rviz" pkg="rviz" type="rviz" args="-f map" required="true"/>
    <node name="cslibs_ndt_2d" pkg="cslibs_ndt_2d" type="cslibs_ndt_2d_map_loader" output="screen" required="true" clear_params="true"/>
</launch>
